cclib/parser/orcaparser.py

```diff
@@ -2107,10 +2130,13 @@ def _append_scfvalues_scftargets(self, inputfile, line):
             if "Last RMS-Density change" in line:
                 rmsDP_value = float(line.split()[4])
                 rmsDP_target = float(line.split()[7])
-            else:
+            elif not '**** THE GBW FILE WAS UPDATED' in line:
                 rmsDP_value = self.scfvalues[-1][-1][2]
                 rmsDP_target = self.scftargets[-1][2]
                 assert deltaE_target == self.scftargets[-1][0]
                 assert maxDP_target == self.scftargets[-1][1]
+            else:
+                rmsDP_value = 0.0
+                rmsDP_target = 0.0
```

Comment:
I've hit this for geometry optimization with unrestricted methods, described in #1177 
However, the fix in this PR is not correct, because it influences the parser output where it shouldn't.

A better fix for this is at https://github.com/pzarabadip/aiida-orca/pull/67

Reply:
@berquist - sorry, I just realized you had replied. Daniel is right, I added this fix as a "quick patch" to avoid an error I was having when using the mini keyword, but his solution seems more robust.

Reply:
This is an example file where I find the issue when parsing with cclib
[Bi2PyrH_SMD_DLPNO.zip](https://github.com/cclib/cclib/files/10764099/Bi2PyrH_SMD_DLPNO.zip)

Reply:
Thanks! Can you add this as a PR to https://github.com/cclib/cclib-data? We'll merge that after confirming any fix to the code gets it parsing.